.env.dist

```diff
@@ -1,17 +1,20 @@
-BOT_TOKEN=123456:Your-TokEn_ExaMple
-ADMINS=123456,654321
-USE_REDIS=False
+# For TgBot
+TGBOT_TOKEN=123456:Your-TokEn_ExaMple
+TGBOT_ADMIN_IDS=[123456,7891011]
+TGBOT_USE_REDIS=False
 
-#POSTGRES_USER=someusername
```

Comment:
This looks good! However, please update the environment section in the Docker Compose file for the PostgreSQL service to reflect these exact variable names for database initialization. We need to ensure these new variable names are appropriately mapped to the old ones used by the image.

Reply:
for this part, I couldnt spot any related references of dotenv for PostgresSQL. If there is any please mention it.